bookwyrm/templates/directory/user_card.html

```diff
@@ -18,7 +18,7 @@
             </div>
         </div>
 
-        <div>
+        <div class="preserve-whitespace">
```

Comment:
In this case, it makes sense to force-wrap the whitespace, since it's a limited preview in a card that is meant to be a standard size. A bio with a lot of line breaks could produce results like:

<img width="324" alt="Screen Shot 2021-07-31 at 10 41 29 AM" src="https://user-images.githubusercontent.com/1807695/127748210-76e005f6-daba-435b-abc6-a232f83bb494.png">
 instead of:
<img width="316" alt="Screen Shot 2021-07-31 at 10 41 49 AM" src="https://user-images.githubusercontent.com/1807695/127748207-09630301-f43b-46a8-82df-cca6716953ab.png">

Since it's a preview, I think it's worth sacrificing the accuracy of the bio display.

Reply:
I ended up adding a new CSS tag "display-inline" since putting multiple blank lines creates a new <p> tag, adding an additional newline character. This could be abused in the current implementation as well, you can see here: https://bookwyrm.social/search?q=EvilDrPurple&type=user
By forcing the user cards to display inline, it will fix this workaround in the current implementation.

Reply:
oh good point!